oe_theme.libraries.yml

```diff
@@ -36,18 +25,6 @@ context_nav:
   dependencies:
     - core/drupal
     - oe_theme/base
-file_translations:
-  js:
-    js/file_translations.js: {}
-  dependencies:
-    - core/drupal
-    - oe_theme/base
-inpage_navigation:
```

Comment:
The library was removed but its still being loaded in paragraph--oe-content-row--variant-inpage_navigation.html.twig

Reply:
Very good catch.